Address review comments in `Content-based message routing` and `Transforming message formats` tutorials (#8248)
ballerina-platform/ballerina-dev-website: swan-lake/integration-tutorials/transforming-message-formats.md

    ![Open diagram view](/learn/images/integration-tutorials/transforming-message-formats/open_diagram_view.gif)

> 3. Generate record types corresponding to the response from the hospital service by providing samples of the expected JSON payload.

Reviewer: ```suggestion
3. Generate record types corresponding to the response from the hospital service by providing a sample of the expected JSON payload.
```

> 3. Generate record types corresponding to the payloads from the hospital service by providing samples of the expected JSON payloads.

Reviewer: There is only one backend call/response/payload here?

Author: Yeah. Anyway I changed this to `Generate record types corresponding to the response from the hospital service by providing samples of the expected JSON payload`

        boolean confirmed;
        string appointmentDate;
    };
    ```

>     Similarly, generate the `HealthcareReservation` and `HospitalReservation` record types corresponding to the parameter and return types of the transforming function. Remove a one `Patient` record if it gets generated twice.

Reviewer: Update to emphasize request/response + where from since the name isn't fixed.

>     Similarly, generate the `HealthcareReservation` and `HospitalReservation` record types corresponding to the request payload, parameter and return types of the transforming function. Remove the duplicate `Patient` record if the same type has already been generated.

Reviewer: Need to update similar to the other one, emphasizing payload rather than name.

>     Similarly, generate the record types corresponding to the request payload, parameter and return types of the transforming function (e.g. `HealthcareReservation`, `HospitalReservation`). Remove the duplicate `Patient` record if the same type has already been generated.

Reviewer: Isn't the request payload the same as the parameter of the transforming function?

Anyway, as mentioned previously, saying transforming function here doesn't really make sense to the user. We need to describe these in terms of request/response payloads.

So

```suggestion
    Similarly, generate the record types corresponding to the request payloads to the healthcare service being developed (e.g. `HealthcareReservation`) and the request payload expected by the backend hospital service (e.g.,`HospitalReservation`). The transformation will be done from the former type to the latter. Also remove the duplicate `Patient` record if the same type has already been generated.
```

Author: +1. Thanks

    };
    ```

    > **Note:**

>     > Since `ssn` must definitely be an array of three integers, make it a fixed-length [array](https://ballerina.io/learn/by-example/arrays/).

Reviewer: ```suggestion
    > Since `ssn` is required to be an array of three integers, make it a fixed-length [array](https://ballerina.io/learn/by-example/arrays/).
```

>     > Since `ssn` must definitely be an array of three integers, make it as a fixed-length [array](https://ballerina.io/learn/by-example/arrays/).

Reviewer: ```suggestion
    > Since `ssn` must definitely be an array of three integers, make it a fixed-length [array](https://ballerina.io/learn/by-example/arrays/).
```

    - Use the `is` check to check whether the response is `ReservationResponse` and return it as is (i.e., reservation successful).

>     - If the response is not a `ReservationResponse` record, log the information at `ERROR` level. Return a `http:NotFound` response if the response from the hospital service is an `http:ClientRequestError` response or an `http:InternalServerError` response otherwise.

Reviewer: ```suggestion
    - If the response is not a `ReservationResponse` record, log the information at `ERROR` level. Return an `http:NotFound` response if the response from the hospital service is an `http:ClientRequestError` response or an `http:InternalServerError` response otherwise.
```
